requirements.txt

```diff
@@ -0,0 +1,5 @@
+matplotlib
```

Comment:
Could you please not include requirements.txt for the PR?
We deleted them on purpose so that users will not just install sth wrong via `pip install -r requirements.txt`